### DISCUSSION TITLE
Precomputed Building Footprint Area and/or Total Floorspace Features?

### DISCUSSION THREAD

**Discussion by SebKrantz** [OvertureMaps/data#189] [Discussion]
It would be very nice to have precomputed features for the building footprint area and the total floorspace (in $m^2$). Many analytical uses require this information and I believe fewer analytical uses require the full building geometry. Precomputing these features could thus save a lot of time and computing resources.

**Comment by jenningsanderson** [comment]
Attributes like length, area, and other summary statistics can be computed from the data itself, and are less valuable to the core _schema_. The exception, of course, is the `bbox`, which is used to filter and access the data. 

That said, these attributes are extraordinarily valuable to specific use cases. This is similar to the request that Overture offer data partitioned by Country. For now, Overture releases the data in just _one_ way, and encourages users to build from it — such as repartitioning or precomputing other attributes that would extend the schema for any specific use case. For example, Fused.io consumes Overture data and repartitions it to be more optimized for their platform (which includes extending the metadata) and hosts it on source.coop.
